pkg/convert/pot.go

```diff
@@ -337,14 +345,16 @@ func writeTags(b *bytes.Buffer, message model.Message) error {
 }
 
 // convertPluralsToMessageString converts a slice of strings to MessageFormat plural form.
-func convertPluralsToMessageString(plurals []string) string {
+func convertPluralsToMessageString(plurals []string) (string, error) {
```

Comment:
Maybe could simplify the function name e.g. messageFromPlurals()?

Reply:
![image](https://github.com/translate-agent/translate/assets/69954124/795a1841-5601-403e-af98-41e5bc176a52)